src/scheduler/resource_resv.c

```diff
@@ -1469,15 +1469,16 @@ remove_resresv_from_array(resource_resv **resresv_arr,
  *
  * @param[in]	resresv_arr	-	job array to add job to
  * @param[in]	resresv	-	job to add to array
- * @param[out]  index -	    Index where resresv is added in resresv_arr
+ * @param[in]	flag -	    flag to indicate whether to set index where the
+ *			    resresv was added in array to resresv itself.
```

Comment:
Make this 'flags' and then instead then list the flag.  Check add_resource_list() for an example.